src/firewall/core/nftables.py

```diff
@@ -237,11 +238,26 @@ def __init__(self, fw):
         self.nftables.set_echo_output(True)
         self.nftables.set_handle_output(True)
 
+    @staticmethod
```

Comment:
The value is that it makes it obvious that the method does not modify the object's state.

The biggest problem with object oriented programming is that while objects encapsulate state, there still is state all over the place and it's unclear when it gets modified. A const method (in C++ terms) is great, because it promises not to modify state. A `@staticmethod` achieves something similar.

Especially, when you have large methods with a self pointer, and the self object is non-trivial. It's entirely non-obvious, whether the method will not modify the object. A `@staticmethod` makes that obvious.